recipes/LibriSpeech/ASR/seq2seq/train_with_wav2vec.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env/python3
+"""Recipe for training a sequence-to-sequence ASR system with librispeech.
+The system employs an encoder, a decoder, and an attention mechanism
```

Comment:
There is no attention mechanism right ? Also the recipe should be teleported to /CTC and not /seq2seq, right?

Reply:
I will change the descriptions. Makes sense, let me move it to `/ctc`.